lib/keys_stringifier.rb

```diff
@@ -0,0 +1,8 @@
+class KeysStringifier
```

Comment:
Please move it into `lib/jsonapi_errors_handler` folder and namespace it in the module